bitcoin/src/blockdata/witness.rs

```diff
@@ -227,6 +229,7 @@ fn resize_if_needed(vec: &mut Vec<u8>, required_len: usize) {
 }
 
 impl Encodable for Witness {
+    // `self.content` includes the varints so encoding here includes them, as expected.n
```

Comment:
Trailing `n`.

Reply:
Glad we don't work in an office together, I'd have to buy you a drink for that one.